Src/.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "dotnet-t4-fix": {
+      "version": "2.3.0-preview-0000-gb1ee03d9bc",
```

Comment:
This should be updated to use a stable release of dotnet-t4 once a new release is available. I ran into errors with the most recent stable release so had to switch to this pre-release version.